.github/workflows/pull-request.yml

```diff
@@ -1,94 +0,0 @@
-name: Pull Request
-
-on:
-  push:
-    branches: '**'
-
-defaults:
-  run:
-    shell: bash
-
-jobs:
-  clippy:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v2
-
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable 
-          profile: minimal
-          components: clippy
- 
-      - name: Clippy
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: --manifest-path rust/Cargo.toml
-
-  rustfmt:
```

Comment:
Why no fmt check for the rust code?

Reply:
Hmm, I don't think rustfmt is necessary for ci, but we can add `yarn run format:check` for that. Because `yarn run format:check` runs `rustfmt` in programs workspace.

Reply:
Imo I think it's good if we are running prettier for the js, soft opinion though